File: backend/internal/model/member.go
package model

type Member struct {
	ID       string `json:"id"`
	Name     string `json:"name"`
	Role     string `json:"role"`
	Bio      string `json:"bio"`
	Photo    string `json:"photo"`
	LinkedIn string `json:"linkedin,omitempty"`
	Twitter  string `json:"twitter,omitempty"`
}

File: frontend/src/types/member.ts
export interface Member {
  id: string;
  name: string;
  role: string;
  teamRole?: string;
  team?: "backend" | "devops" | "qa";
  bio: string;
  initials: string;
  avatarColor: string;
  image?: string;
  linkedin?: string;
  twitter?: string;
  github?: string;
  isFounder?: boolean;
  skills?: string[];
  experience?: {
    company: string;
    role: string;
    highlights: string[];
  }[];
  projects?: {
    name: string;
    description: string;
    tech: string[];
  }[];
}
